tests/Support/Policies/RedPolicyWithMessage.php

```
@@ -0,0 +1,87 @@
+     */
+    public function view($user, Model $model)
+    {
+        return Response::deny('You don\'t have permission to view user');
+    }
+
+    /**
+     * Determine whether the user can create models.
+     *
+     * @param $user
+     */
+    public function create($user)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$user'. (undefined)

(UnusedFormalParameter)